public/defaults/report_watcher.json

```diff
@@ -6,6 +6,19 @@
     "schedule": {
       "later": "every 1 hour"
     }
+  },
```

Comment:
Why do you add `input` and `condition` in the report action? The goal of the report is to take a screenshot of a web page. If you want to execute another action based on some condition in the same watcher, just add a new action under the same watcher.
![screenshot from 2018-03-15 16-05-08](https://user-images.githubusercontent.com/5389745/37471804-e70e6866-286a-11e8-9fc4-4c883d625cf1.png)
![screenshot from 2018-03-15 16-05-25](https://user-images.githubusercontent.com/5389745/37471816-ee7b2120-286a-11e8-9e35-7f9608386ffe.png)